README.md

```diff
@@ -1,5 +1,9 @@
 # DevOps-Apprenticeship-Notes
+k-6-youtube-workshops
+* Here is the docker post-install instructions that reduce the number of sudo commands: https://docs.docker.com/engine/install/linux-postinstall/
 
+*  adding some tfswitch vedio. Here is the tfswitch video link: https://drive.google.com/file/d/14xNJzpCajLEVzcGL5h3twfIyD-hRhkGc/view?usp=sharing
```

Comment:
I would suggest fixing this spelling error and/or making this one sentence. 
```suggestion
* Here is the tfswitch video link: https://drive.google.com/file/d/14xNJzpCajLEVzcGL5h3twfIyD-hRhkGc/view?usp=sharing
```

When doing Pull Requests, feel free to take a look at your own submitted code/documentation and see if there is any other spelling/grammar issues.